semantic-convention-utils/src/test/java/org/hypertrace/semantic/convention/utils/http/HttpSemanticConventionUtilsTest.java

```diff
@@ -331,7 +331,7 @@ public void testGetHttpRequestHeaderPath() {
         createMockEventWithAttribute(
             RawSpanConstants.getValue(HTTP_REQUEST_HEADER_PATH), "sample/http/request/header/path");
     assertEquals(
-        Optional.of("sample.http.request.header.path"),
+        Optional.of("sample/http/request/header/path"),
```

Comment:
how come it was passing earlier?

Reply:
It was due to sanitize method which is now moved to traceable side. So modified the test a bit.

Reply:
wait, but thats changing the business logic of the method. Are there no tests for the methods which use this method `HttpSemanticConventionUtils.getHttpRequestHeaderPath(event)` (if yes, then please add a test)?

Reply:
It is a private method called from some other method so it would be tested with the parent method

Reply:
Also made the corresponding changes on traceable side